webcompat/helpers.py

```diff
@@ -84,11 +84,42 @@ def md5_checksum(file_path):
         return m.hexdigest()
 
 
-def get_str_value(val):
-    """Map values from JSON to python."""
+def get_list_items(val_dict):
+    """Return list items (<li>) from the passed in list ([])."""
+    rv = ''.join(['<li>{k}: {v}</li>'.format(k=k, v=get_serialized_value(v))
+                 for k, v in list(val_dict.items())])
```

Comment:
The code was inherited from probably the conversion from python 2 to python 3. IN this specific case `val_dict.items()` is a `<class 'dict_items'>` and is an iterable, which makes the `list()` wrapper unecessary